enterprise_access/settings/devstack.py

```diff
@@ -59,3 +59,5 @@
 )
 # END CELERY
 
+ECOMMERCE_URL = 'http://ecommerce.example.com'
```

Comment:
Are there devstack URLs that would actually work we could put in here?

Reply:
actually kinda curious, if you run tests locally, does it say it's using devstack.py as the env? i found locally that test env was being ignored (and using devstack instead). so i put those there to get tests to work. but yeah, i can change these once i get the tests running against the right env

Reply:
I did not try running, only read the code.

Reply:
alrighty, i'm inclined to file a follow up ticket/make a follow up PR to change this value in devstack.py. this PR has enough stuff in it ><'

agreed though that devstack.py and test.py should have different values

Reply:
Follow up ticket on this is fine.

Reply:
ticket created